toolchain/c-toolchain.lisp

```diff
@@ -223,12 +218,24 @@
           *linkkit-start* nil
           *linkkit-end* nil)))
 
+(defun common-toolchain-parameters ()
+  (setf *cc-flags*
+        (append
+         (split-cflags (getenv "CFLAGS"))
+         *cc-flags*
+         ;; For MacPorts
+         #+darwin (list "-I" "/opt/local/include/")
+         ;; FreeBSD non-base header files
+         #+(or freebsd openbsd) (list "-I" "/usr/local/include/"))))
+
 (defun ensure-toolchain-parameters ()
-  #+clisp (unless *cc* (clisp-toolchain-parameters))
-  #+ecl (unless *cc* (ecl-toolchain-parameters))
-  #+mkcl (unless *cc* (mkcl-toolchain-parameters))
-  #+sbcl (unless *cc* (sbcl-toolchain-parameters))
-  (unless *cc* (default-toolchain-parameters)))
+  (unless *cc*
+    #+clisp (clisp-toolchain-parameters)
+    #+ecl (ecl-toolchain-parameters)
+    #+mkcl (mkcl-toolchain-parameters)
+    #+sbcl (sbcl-toolchain-parameters)
+    (unless *cc* (default-toolchain-parameters))
```

Comment:
Note that `(sbcl-toolchain-parameters)` does not set `*cc*` on Travis. Here is the check where `(unless *cc* (default-toolchain-parameters))` was replaced by `#-(or clisp ecl mkcl sbcl) (default-toolchain-parameters)`: https://travis-ci.org/cffi/cffi/jobs/575204326